packages/helix-shared-bounce/package.json

```diff
@@ -30,6 +30,8 @@
     "@adobe/helix-universal": "1.9.1"
   },
   "devDependencies": {
-    "nock": "13.2.1"
-  }
+    "nock": "13.2.1",
+    "proxyquire": "2.1.3"
+  },
+  "engines" : { "node" : ">=15.0" }
```

Comment:
why?

(if a higher version than 14 is needed, then please use 16)

Reply:
Node 15 is the earliest version that supports timers, but it is not yet supported by lambda, so I need to change this anyway.

Reply:
node 15 no longer needed.